### DISCUSSION TITLE
[Q&A] windows 本地开发，如何安装依赖

### DISCUSSION THREAD

**Discussion by JZpretend** [lobehub/lobehub#402] [Development Question] [answered]
### 🧐 问题描述 | Proposed Solution

我用npm install 出现这个错误
(node:8276) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 close listeners added to [TLSSocket]. Use emitter.setMaxListeners() to increase limit
Bun 好像win系统安装不了
求各位大佬解答！

**Comment by canisminor1990** [comment, marked as answer]
win下用 pnpm
